src/overlays/actors/ovl_Bg_Dodoago/z_bg_dodoago.c

```diff
@@ -33,23 +35,259 @@ const ActorInit Bg_Dodoago_InitVars = {
     (ActorFunc)BgDodoago_Update,
     (ActorFunc)BgDodoago_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Dodoago/BgDodoago_SetupAction.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Dodoago/func_80871A08.s")
+static ColliderCylinderInit sColCylinderInit0 = {
+    { COLTYPE_UNK10, 0x00, 0x39, 0x00, 0x00, COLSHAPE_CYLINDER },
+    { 0x02, { 0x00000000, 0x00, 0x00 }, { 0xFFCFFFFF, 0x00, 0x00 }, 0x00, 0x01, 0x00 },
+    { 80, 30, 80, { 0, 0, 0 } },
+};
+
+static ColliderCylinderInit sColCylinderInit1 = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x3D, 0x20, COLSHAPE_CYLINDER },
+    { 0x02, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 50, 60, 280, { 0, 0, 0 } },
+};
+
+s16 D_80872598 = 0x00000000;
+
+Color_RGBA8_n D_8087259C = { 100, 100, 100, 0 };
+Color_RGBA8_n D_808725A0 = { 40, 40, 40, 0 };
+
+Vec3f VELOCITY = { 0.0f, -1.5f, 0.0f };
+Vec3f ACCELERATION = { 0.0f, -0.20000000298023223876953125f, 0.0f };
```

Comment:
I wonder if these variables are in-function static (in `func_80871A08`).
```suggestion
Color_RGBA8_n D_8087259C = { 100, 100, 100, 0 };
Color_RGBA8_n D_808725A0 = { 40, 40, 40, 0 };

static Vec3f sVelocity = { 0.0f, -1.5f, 0.0f };
static Vec3f sAcceleration = { 0.0f, -0.2f, 0.0f };
```